spec/bolt/transport/ssh_spec.rb

```diff
@@ -14,9 +14,11 @@
   include BoltSpec::Files
   include BoltSpec::Task
 
+  let(:boltdir) { Bolt::Boltdir.new('') }
```

Comment:
What's the difference between `Bolt::Boltdir.new('')` and `Bolt::Boltdir.new('.')`?

Reply:
One creates an empty base path and one creates a base path of `.`

In practice, I don't think there's a difference, because empty string + "/foo" is the same as "./foo"

Reply:
I'd hope for either an explanation around the expectations, or to use a consistent default to mean "we don't care".